packages/@expo/cli/src/start/server/platformBundlers.ts

```diff
@@ -1,15 +1,28 @@
 import { ExpoConfig, Platform } from '@expo/config';
+import resolveFrom from 'resolve-from';
 
 /** Which bundler each platform should use. */
 export type PlatformBundlers = Record<Platform, 'metro' | 'webpack'>;
 
 /** Get the platform bundlers mapping. */
-export function getPlatformBundlers(exp: Partial<ExpoConfig>): PlatformBundlers {
+export function getPlatformBundlers(
+  projectRoot: string,
+  exp: Partial<ExpoConfig>
+): PlatformBundlers {
+  /**
+   * SDK 50+: The web bundler is dynamic based upon the presence of the `@expo/webpack-config` package.
+   */
+  let web = exp.web?.bundler;
+  if (!web) {
+    const resolved = resolveFrom.silent(projectRoot, '@expo/webpack-config/package.json');
```

Comment:
interesting. be sure to update the schema description in universe to indicate how the new default value works.

Reply:
https://github.com/expo/universe/pull/14308